src/UI/Components/SingleTextSelect/SingleTextSelect.tsx

```diff
@@ -114,15 +114,14 @@ export const SingleTextSelect: React.FC<Props> = ({
       ]);
     } else {
       setSelectOptions(options);
+
+      if (inputValue && inputValue === selected) {
+        setInputValue(getDisplayValue(selected));
+      }
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [options]);
```

Comment:
isn't selected here required? I'm out of the flow here so if the result is correct then it's fine :)

Reply:
tested and indeed, also works. I had thought it would chain renders but it's fine.